e2e/.env-example

```diff
@@ -0,0 +1,4 @@
+# Go to https://automate.browserstack.com/dashboard and copy the username and
+# access key. Then, rename this file to ".env".
```

Comment:
I would suggest here copying the file instead of renaming it, or we could end up removing the `.env-example` file by accident.

Reply:
Done 👍